### PR TITLE
Apply max render length to evaluate_terraform.py

### PR DESCRIPTION
**Pull request by nimrodkor**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by gruebel — APPROVED** [review]
🌃

File: checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py [hunk 1]
```diff
@@ -23,9 +26,15 @@
 DIRECTIVE_EXPR = r"\%\{([^\}]*)\}"
 
 COMPARE_REGEX = re.compile(r"^(?P<a>.+)(?P<operator>==|!=|>=|>|<=|<|&&|\|\|)+(?P<b>.+)$")
+CHECKOV_RENDER_MAX_LEN = force_int(os.getenv("CHECKOV_RENDER_MAX_LEN", "10000"))
 
 
 def evaluate_terraform(input_str: str, keep_interpolations: bool = True) -> Any:
+    if CHECKOV_RENDER_MAX_LEN and 0 < CHECKOV_RENDER_MAX_LEN < len(input_str):
```
Comment (gruebel) [inline checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py:33]:
```suggestion
    if isinstance(input_str, str) and CHECKOV_RENDER_MAX_LEN and 0 < CHECKOV_RENDER_MAX_LEN < len(input_str):
```
maybe?

Comment (nimrodkor) [reply checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py:33]: It's practically called `input_str`, and it's of type `str` (see the line above, line 32)

Comment (nimrodkor) [reply checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py:33]: Just saw the tests 🤦

**Pull request merged**